config.json

```
@@ -1239,7 +1239,38 @@
       "name": "while loops"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Modern",
+      "content": "Swift - result of the latest research on programming languages, combined with of Apple experience.",
```

I made some small tweaks to fix "combined with of" (which is missing some words) and to have the result fit in 100 characters.

```suggestion
      "content": "Swift incorporates the latest research on programming languages, combined with Apple's experience.",
```